src/CalcViewModel/Common/NavCategory.cpp

```diff
@@ -89,10 +89,10 @@ bool IsGraphingModeEnabled()
     DWORD bufferSize{ sizeof(allowGraphingCalculator) };
     // Make sure to call RegGetValueW only on Windows 10 1903+
     if (RegGetValueW(
-            HKEY_LOCAL_MACHINE,
+            HKEY_CURRENT_USER,
```

Comment:
We should probably check both, a group policy can be defined per machine or per user. I suppose in the case of shared devices, machine policy can be useful.

Reply:
Our policy can only be set per user. I tried to set it per machine using the Group Policy editor and it was not available.